spec/support/vcr.rb

```diff
@@ -41,6 +41,7 @@
   c.filter_sensitive_data('<MULESOFT_SECRET>') { Settings.form_10_10cg.carma.mulesoft.client_secret }
   c.filter_sensitive_data('<SHAREPOINT_CLIENT_SECRET>') { Settings.vha.sharepoint.client_secret }
   c.filter_sensitive_data('<ADDRESS_VALIDATION>') { VAProfile::Configuration::SETTINGS.address_validation.hostname }
+  c.filter_sensitive_data('<X_API_KEY>') { Settings.mhv.rx.x_api_key }
```

Comment:
I don't see this used in any cassettes. Will it be used later or should it be deleted?